public/app/plugins/datasource/loki/datasource.ts

```diff
@@ -102,6 +102,38 @@ export class LokiDatasource extends DataSourceApi<LokiQuery, LokiOptions> {
     return getBackendSrv().fetch<Record<string, any>>(req);
   }
 
+  getLogsVolumeQuery(request: DataQueryRequest<LokiQuery>): Observable<DataQueryResponse> | undefined {
+    const isLogsVolumeAvailable = request.targets.some((target) => !isMetricsQuery(target.expr) && !!target.expr);
+    if (!isLogsVolumeAvailable) {
+      return;
+    }
+
+    const histogramRequest = cloneDeep(request!);
```

Comment:
```suggestion
    const histogramRequest = cloneDeep(request);
```

is the `!` needed here? it seems to me that `request` cannot be nullish.

Reply:
Fixed in #39042